Added new svg logo for the app

imaryandokania opened the pull request:
## Pull Request template


## Purpose / Description
I have converted a png to svg.
![images](https://user-images.githubusercontent.com/61244704/111402489-63888b80-86f3-11eb-8649-48b5d63c9870.jpeg)

This is added for future drawable additions.

## Approach
The svg is licensed under MIT license and Is compatible with GPL.

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

Arthur-Milchior commented:
Hi

Thanks for the commit. I thought we already had a svg. I think @david-allison-1 followed this more closely than myself. In any case, thanks for converting it.

There are four commits in this PR. Not a huge trouble as I can squash. But it would be helpful for reviewer if you can limit the amount of commits, so that I don't spend time looking as useless data.

You may want in particular to get the habit of rebasing onto master instead of merging

david-allison-1 commented on docs/graphics/src/logo.svg:
```diff
@@ -0,0 +1,32 @@
+<!--
+MIT License
+
+Copyright (c) [2021] [Anki Droid]
```
> The svg is licensed under MIT license and Is compatible with GPL.

Can you please confirm (ideally in public) that the author has explicitly agreed for the file to be licensed as such and provide copyright via either their name/web handle

imaryandokania replied:
<img width="592" alt="Screenshot 2021-03-17 at 10 19 51 AM" src="https://user-images.githubusercontent.com/61244704/111415596-562acb80-870a-11eb-8d97-21ceca103316.png">
He is the creator!

david-allison-1 replied:
Sorry to be annoying, could you explicitly confirm that he's OK with licensing it under the MIT License for us.

david-allison-1 reviewed (changes requested):
See comment - we need to be very careful about assigning licenses to files that we don't own.

I personally asked about using the icon a while back, so I'm expecting this'll be fine, but we need to be on the right side of the law

imaryandokania commented:
> See comment - we need to be very careful about assigning licenses to files that we don't own.
> 
> I personally asked about using the icon a while back, so I'm expecting this'll be fine, but we need to be on the right side of the law

I have replied in discord as well that the creator has replied and was okay if we use the image , if you want I can share the conversation with the creator the logo .

imaryandokania commented:
Yes I have messaged the creator regarding license as well .. waiting for
his reply

On Wed, 17 Mar 2021 at 10:58 AM David Allison ***@***.***>
wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In docs/graphics/src/logo.svg
> <https://github.com/ankidroid/Anki-Android/pull/8251#discussion_r595717621>
> :
>
> > @@ -0,0 +1,32 @@
> +<!--
> +MIT License
> +
> +Copyright (c) [2021] [Anki Droid]
>
> Sorry to be annoying, could you explicitly confirm that he's OK with
> licensing it under the MIT License for us.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ankidroid/Anki-Android/pull/8251#discussion_r595717621>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOTIKIH455EOFOSMJQLXRVLTEA26FANCNFSM4ZJVEQPA>
> .
>

Arthur-Milchior commented:
Ideally, if you can ask the author to confirm here, would allow to keep track and avoid to use you as a middle-man in case of need

imaryandokania commented:
> Ideally, if you can ask the author to confirm here, would allow to keep track and avoid to use you as a middle-man in case of need

Okay I will tell him to comment it here

imaryandokania commented:
I have rebased this, so won't be a problem now

lucajulius commented:
Hi, I'm the original author of the logo and I'm happy to license it to AnkiDroid under the MIT license.

david-allison-1 commented on docs/graphics/src/logo.svg:
```diff
@@ -0,0 +1,32 @@
+<!--
+MIT License
+
+Copyright (c) [2021] [Anki Droid]
```
```suggestion
Copyright (c) 2021 lucajulius
```

david-allison-1 reviewed (changes requested):
Thanks!!!! Let's get this in

imaryandokania commented:
> Thanks!!!! Let's get this in

Yeah !

david-allison-1 reviewed (approved):
LGTM. Requires squash merge

imaryandokania commented:
> LGTM. Requires squash merge

do I rebase it again?

david-allison-1 commented:
> do I rebase it again? So that you don't need to squash merge

It'd be appreciated. You can perform a squash as part of a rebase.

Do try and inform maintainers if you're making major changes after an "accepted" review

imaryandokania commented:
@david-allison-1 It is rebased.Please review again

imaryandokania commented:
> > do I rebase it again? So that you don't need to squash merge
> 
> It'd be appreciated. You can perform a squash as part of a rebase.
> 
> Do try and inform maintainers if you're making major changes after an "accepted" review

I have rebased and also added comment for review

david-allison-1 reviewed (approved):
Still looks good!

mikehardy reviewed (approved):
did not realize this was waiting on a second approval sorry :-)
Thank you!

nwwt commented:
Two questions to add, hope it's ok to add them after already merging…

* The file is huge for a svg + rather simple geometric shapes (almost 10x anki.svg, despite that being more complex)? Probably because the lines are wobbly.
* Shouldn't the outside of a circle be transparent? It'll stick out on non-white backgrounds.

Arthur-Milchior commented:
@nwwt 
The first point do not seems to be a question. If you can make a SVG as nice and with lower size, that'd be great. 
If you know how to replace while by transparent background it would be nice. I clearly missed that, thanks fo pointi this out.
